symengine/printer.cpp

```diff
@@ -712,7 +712,7 @@ void StrPrinter::bvisit(const MultivariatePolynomial &x)
                     expr << "*";
                 }
                 expr << it->__str__();
-                if (exps[i] > 1)
+                if (exps[i] > 1 or exps[i] < 0)
```

Comment:
Yes, I will. But it's a correction for negative exponents in the Expression polynomials. those don't get printed, in the previous code.